### PR TITLE
Add help, label, and edit form changes to config.

### PR DESCRIPTION
**Pull request by beeyayjay**
## Description
Contains help text edits etc from onhelptext demo environment

See #1828

## QA steps
- Check that help text, labels, and edit forms appear as expected.
- Double-check that only appropriate config files have been updated.

### CODE REVIEW

**Comment by kevwalsh** [comment]
I'll update behat tests and force push!

**Comment by kevwalsh** [comment]
Behat tests are fixed in 
https://github.com/beeyayjay/va.gov-cms/pull/1

File: config/sync/field.field.node.event_listing.field_intro_text.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ id: node.event_listing.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: event_listing
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.event_listing.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.health_care_region_detail_page.field_intro_text.yml [hunk 2]
```diff
@@ -9,8 +9,8 @@ id: node.health_care_region_detail_page.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: health_care_region_detail_page
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.health_care_region_detail_page.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.health_care_region_page.field_intro_text.yml [hunk 3]
```diff
@@ -9,8 +9,8 @@ id: node.health_care_region_page.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: health_care_region_page
-label: 'Intro text'
-description: 'Note: If the VAMC has an official system name, use it in the intro text. If not, use the VAMC system name.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.health_care_region_page.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.health_services_listing.field_intro_text.yml [hunk 4]
```diff
@@ -9,8 +9,8 @@ id: node.health_services_listing.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: health_services_listing
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.health_services_listing.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.landing_page.field_intro_text.yml [hunk 5]
```diff
@@ -9,8 +9,8 @@ id: node.landing_page.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: landing_page
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information. This field supports the following HTML tags: <code>&#x3C;em&#x3E;&#x3C;strong&#x3E;&#x3C;br&#x3E;&#x3C;p&#x3E;&#x3C;a&#x3E;</code>.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.landing_page.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.leadership_listing.field_intro_text.yml [hunk 6]
```diff
@@ -9,8 +9,8 @@ id: node.leadership_listing.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: leadership_listing
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.leadership_listing.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.locations_listing.field_intro_text.yml [hunk 7]
```diff
@@ -9,8 +9,8 @@ id: node.locations_listing.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: locations_listing
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.locations_listing.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.page.field_intro_text.yml [hunk 8]
```diff
@@ -9,8 +9,8 @@ id: node.page.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: page
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information. This field supports the following HTML tags: <code>&#x3C;em&#x3E;&#x3C;strong&#x3E;&#x3C;br&#x3E;&#x3C;p&#x3E;&#x3C;a&#x3E;</code>.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.page.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

File: config/sync/field.field.node.press_releases_listing.field_intro_text.yml [hunk 9]
```diff
@@ -9,8 +9,8 @@ id: node.press_releases_listing.field_intro_text
 field_name: field_intro_text
 entity_type: node
 bundle: press_releases_listing
-label: 'Intro text'
-description: 'This is displayed directly beneath the title as an introduction to the page. This text should help a user figure out whether the information on this page is relevant to them, and should be written in the 2nd person. It will never be shown alongside the Description, so it''s ok for it to be duplicative of that information.'
+label: 'Page introduction'
+description: 'Add an introduction that helps visitors understand if information on the page is relevant to them. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.press_releases_listing.field_intro_text.yml:13]:
```suggestion
description: 'Add an introduction that helps visitors understand if information on the page is relevant to them.'
```

**Pull request merged**